starfish/experiment/builder/__init__.py

```diff
@@ -89,9 +89,10 @@ def build_image(
             ImageFormat.TIFF,
         )
 
-        for z_ix in range(z_count):
-            for round_ix in range(round_count):
-                for ch_ix in range(ch_count):
+        # order reflects the most common acquisition order of the data
```

Comment:
https://github.com/spacetx/starfish/pull/758

Reply:
Nice, thanks for doing this! I made a comment.

Reply:
Landed, you can now rebase. :)